org.eclipse.winery.workflowmodeler/mock/partition/index.js

```diff
@@ -0,0 +1,2 @@
+module['exports'] = function (server) {
+};
```

Comment:
Could you try to add a `.editorconfig` so that each file ends with a new line.

Here a proposal for an .editorconfig, placed in `org.eclipse.winery.workflowmodeler`.

```
[*]
charset = utf-8
indent_style = space
indent_size = 4
insert_final_newline = true
trim_trailing_whitespace = true
```